Sources/Datadog/DatadogInternal/Context/DatadogContext.swift

```diff
@@ -56,6 +56,9 @@ internal struct DatadogContext {
     /// Current device information.
     let device: DeviceInfo
 
+    /// Current user information.
+    var userInfo: UserInfo
+
```

Comment:
Question: why do we need this and some other`DatadogContext` properties to be mutable (`var`) 🤔💭?

Reply:
So they can be mutated by the publishers in the [context provider](https://github.com/DataDog/dd-sdk-ios/blob/feature/v2/Sources/Datadog/DatadogCore/Context/DatadogContextProvider.swift#L99). But since the `DatadogContext` is a struct, there is no way of mutating those values outside of the provider.

Reply:
Ah, got it 👍. So it will be `public internal(set) var` at the end - means still immutable for Features.